CONTRIBUTING.md

```diff
@@ -57,6 +57,79 @@ You can build and start a local docs website by running these two commands.
   - `npm run docs-buildDockerImage` OR `docker build -t squidfunk/mkdocs-material ./docs/`
   - `npm run docs-runLocalDocker` OR `docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material`
 
+### Tests
+
+Tests are under `tests` folder of each modules and split into two categories: unit tests and e2e tests.
+
+You can run each group separately or all together thanks to [jest-runner-groups](https://www.npmjs.com/package/jest-runner-groups).
+
+Unit tests, under `tests/unit` folder are standard [Jest](https://jestjs.io) tests.
+
+Integration tests under `tests/e2e` folder on the other hand will test the module features by deploying Lambdas functions into your AWS Account. Infrastructure deployment is possible thanks to CDK lib for Typescript and `aws sdk` is used to invoke the functions and assert on the expected behavior. All of this is orchestrated using the same Jest framework used for the unit tests.
```

Comment:
```suggestion
End-to-end tests, under `tests/e2e` folder, will test the module features by deploying Lambdas functions into your AWS Account. We use CDK lib for Typescript for creating infrastructure, and `aws sdk` for invoking the functions and assert on the expected behaviors. All steps are also executed by Jest.
```

Reply:
Change to make it more succinct.  

Let's use the term e2e tests rather than integration testing to avoid ambiguity.